build.gradle

```diff
@@ -253,45 +257,50 @@ project(':embedded-database-spring-test') {
     dependencies {
         api project(':embedded-database-spring-test-autoconfigure')
 
-        api 'org.testcontainers:postgresql:1.18.3'
-        api 'org.testcontainers:mssqlserver:1.18.3'
-        api 'org.testcontainers:mysql:1.18.3'
-        api 'org.testcontainers:mariadb:1.18.3'
+        api 'org.testcontainers:postgresql:1.20.1'
```

Comment:
My intention is to support a wide range of versions at runtime, including the latest ones, but when it comes to default versions, these should be held back a bit. For more details on why I believe using the latest versions in projects like this isn't ideal, check out my previous comment:
https://github.com/zonkyio/embedded-database-spring-test/issues/260#issuecomment-1944950133